deck.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'flashcard'
+require_relative 'card_parser'
+
+class Deck
+  attr_accessor :deck, :flashcards
+
+  include Parsable
```

Comment:
The only place you ever call the method in this module is in the Controller. Because it is never called in this class, you shouldn't mix in this module.

By doing so, you are giving the Deck the ability to read text files and create cards, which it shouldn't necessarily know how to do.